src/lib/y2storage/dasd.rb

```diff
@@ -0,0 +1,50 @@
+# encoding: utf-8
+
+# Copyright (c) [2017] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2storage/storage_class_wrapper"
+require "y2storage/partitionable"
+
+module Y2Storage
+  # A DASD (direct-access storage device), typically used in mainframes
```

Comment:
@jreidinger: Did you have a look at that page you linked? Do you find that documentation useful? It just explains how hard disks work in general; this is very much the same as the documents linked from the DASD Wikepedia page; one is an ancient IBM document from 1975 (!) that also just explains on 27 pages how magnetic disks work and what cylinders, heads and sectors are, the other one from 2015 at least has enough mercy to omit that stuff, but it also does not contain any useful content, in particular no explanation what is the difference between DASDs and other hard disks; much less that even touches the topic of DASDs on Linux.